### DISCUSSION TITLE
Not possible to translate channel selection text

### DISCUSSION THREAD

**Discussion by flo-assling** [globaleaks/globaleaks-whistleblowing-software#3616] [Q&A] [answered]
### What version of GlobaLeaks are you using?

4.13.5

### What browser(s) are you seeing the problem on?

All

### What operating system(s) are you seeing the problem on?

Windows, macOS, iOS

### Describe the issue

When chosing German language, the text "Select a reporting channel:" is not translated.
When adding a custom translation, it does not change.

### Proposed solution

Please make it possible to translate this text snippet :-)

**Comment by evilaliv3** [comment, marked as answer]
Hello, thank you @flo-assling 

Actually translations are handled on transifex: https://app.transifex.com/otf/globaleaks

If you would like to register here, we could grant you the privilege to complete the missing translations

**Reply by flo-assling** [reply]
I signed up and will work on that! :-)

**Comment by flo-assling** [comment]
@evilaliv3 can you please acknowledge my request so I can start with the missing translations? Thanks a lot! :-)

**Comment by flo-assling** [comment]
I don't know if this helps in this case...
I already entered a translation as described, but the text is still shown in english...
![Bildschirmfoto 2023-09-15 um 09 29 13](https://github.com/globaleaks/GlobaLeaks/assets/7966697/f4d88570-b3ba-4e57-a681-775ee63b49fa)
![Bildschirmfoto 2023-09-15 um 09 29 37](https://github.com/globaleaks/GlobaLeaks/assets/7966697/27aa3d1d-f069-419e-9a0d-bb8c66b40cc8)
.

**Comment by evilaliv3** [comment]
Hello Florian,

This settings can be currently be configured in the Advanced settings

**Comment by flo-assling** [comment]
Ah! Worked! Thanks for letting me know :-)
